File: src/BukkitPE/level/generator/populator/Populator.php
<?php
/**
 * All the Object populator classes
 */
namespace BukkitPE\level\generator\populator;

use BukkitPE\level\ChunkManager;
use BukkitPE\utils\Random;

abstract class Populator{
	public abstract function populate(ChunkManager $level, $chunkX, $chunkZ, Random $random);
}

File: src/net/BukkitPE/level/generator/populator/Populator.java
package net.BukkitPE.level.generator.populator;

import net.BukkitPE.level.ChunkManager;
import net.BukkitPE.math.NukkitRandom;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public abstract class Populator {
    public abstract void populate(ChunkManager level, int chunkX, int chunkZ, NukkitRandom random);
}
